lib/iris/plot.py

```diff
@@ -286,7 +367,7 @@ def _draw_2d_from_bounds(draw_method_name, cube, *args, **kwargs):
         string_axes = {}
 
         for coord, axis_name, data_dim in zip([u_coord, v_coord],
-                                              ['xaxis', 'yaxis'],
+                                              ['plotxaxis', 'yaxis'],
```

Comment:
This should be changes back to `'xaxis'`
axis_name gets added to `string_axes` which is used in [_string_coord_axis_tick_labels](https://github.com/corinnebosley/iris/blob/1820e750315b07bdbdd6acadbe77fa23d4fca750/lib/iris/plot.py#L242) where it will try
```getattr(ax, 'plotxaxis')```
which doesn't exist. But
```gettattr(ax, 'axis')```
does